articles/quickstart/webapp/laravel/01-login.md

```diff
@@ -56,41 +56,59 @@ First, we need to add the Auth0 Services to the list of Providers in `config/app
 ```php
 // config/app.php
 
-'providers' => array(
-    // ...
-    Auth0\Login\LoginServiceProvider::class,
-);
+<?php
+
+    return [
+        // ...
+
+      'providers' => [
+          // ...
+          Auth0\Login\LoginServiceProvider::class,
+      ],
 ```
 
 If you want to use an `Auth0` facade, add an alias in the same file (not required, [more information on facades here](http://laravel.com/docs/8.x/facades)):
 
 ```php
 // config/app.php
 
-'aliases' => [
-    // ...
-    'Auth0' => Auth0\Login\Facade\Auth0::class,
-];
+<?php
+
+    return [
+        // ...
+
+      'aliases' => [
+          // ...
+          'Auth0' => Auth0\Login\Facade\Auth0::class,
+      ],
 ```
 
 Finally, you will need to bind a class that provides the app's User model each time a user is logged in or a JWT is decoded. You can use the `Auth0UserRepository` provided by this package or build your own (see the "Custom User Handling" section below).
 
-Add the following to your `AppServiceProvider::register()` method:
+Replace the contents of that file with the following:
```

Comment:
What does `that file` mean here?

Reply:
The file path is indicated in the source code markup that follows that line, app/Providers/AppServiceProvider.php, but I'll reword that line to make it clearer.